appveyor.yml

```diff
@@ -1,14 +1,13 @@
-init:
-  - git config --global core.autocrlf true
-branches:
-  only:
-    - master
-    - release
-    - dev
-    - /^(.*\/)?ci-.*$/
+image: Windows Server 2016
+
+install:
+  - ps: Invoke-WebRequest "https://github.com/docker/compose/releases/download/1.11.2/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\docker\docker-compose.exe
+  - ps: Set-MpPreference -DisableRealtimeMonitoring $true
+
 build_script:
+  - docker-compose -f .\docker-compose.windows.yml up -d
```

Comment:
Yes, that would be a great test if the Docker build really works on Windows. Just wondered after looking at the Dockerfile.windows and realized that it currently uses a Linux base image.